components/Button/index.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import MaterialButton from '@material-ui/core/Button';
+import { styled } from '@material-ui/core/styles';
```

Comment:
We import the default button from `material-ui` but call it `MaterialButton` instead of just `Button`.